docs/_docs/extensions/datagrid.md

```diff
@@ -109,6 +109,26 @@ The grid can work in two different editing modes that can provide different user
 - `Inline` editing is done in the current row
 - `Popup` editing is done in the the modal dialog
 
+### Cascading values
+
+In some case you want to update a different cell in a DataGrid when you update a value. This can be achieved with an `UpdateCell` method. You have two ways of updating a cell:
+
+- by using the `context` inside of `EditTemplate`
+- or by using the `DataGrid` instance
+
+In the following example we're simply calling `context.UpdateCell` with a fieldname to change and a new value that we want it to assign: 
+
+```html
+<DataGridColumn TItem="Employee" Field="@nameof( Employee.Salary )" Caption="Salary" Editable="true">
+    <EditTemplate>
+        <NumericEdit TValue="decimal" Value="@((decimal)context.CellValue)" ValueChanged="@( v => {
+                context.CellValue = v;
+                context.UpdateCell( "Tax", v * .25m );
```

Comment:
Even in docs, show good practice examples if possible. `nameof` for Tax?

Reply:
Makes sense